decision-policy.php

```diff
@@ -14,27 +14,37 @@
 				</ol>
 			</li>
+			<li>Responding to the Call for Consensus<ol>
+					<li>Individuals are invited, but not required, to support the decision by replying to the CfC email with “+1”, or to indicate a “can live with” status by replying with a “+0”.</li>
+					<li>Objections can be raised by replying to the CfC email with “-1”. In order to facilitate understanding and discussion, it is recommended to include a rationale which includes:<ol>
+							<li>why the individual objects to the proposed decision;</li>
+							<li>describes how the objection was raised and considered during discussion leading to the CfC, including how reactions to the potential objection were addressed;</li>
+							<li>what alternate decision would remove the objection <b>and</b> be likely to gain consensus in the WG.</li>
+						</ol></li>
+					<li>Objections are subject to examination by the WG and therefore must be filed on the public record.</li>
+				</ol></li>
 			<li>Evaluating the Call for Consensus. 
 				<ol>
 					<li>If no objections are received by the deadline, the draft decision becomes a formal decision of the Working Group. </li>
-					<li>If objections are received but the chairs believe the objections have already been considered as far as is possible and reasonable, and the reviewers providing the objections can "live with” the decision, the draft decision becomes a formal decision of the Working Group. </li>
-					<li>If objections are received that the chairs believe present substantive new information or if the chairs believe there is not a clear consensus in the Working Group, they will reopen the discussion, as detailed in <a href="http://www.w3.org/Consortium/Process/#WGChairReopen">section 3.3.4 of the Process Document (Reopening a Decision When Presented With New Information)</a>.</li>
-					<li>If working group member(s) continue to disagree and the chairs do not believe it presents substantive new information, or it does not meet the criteria established for adding new normative content, the chairs may decide the draft decision becomes a formal decision of the Working Group despite the objection.</li>
+					<li>If objections are received but the chairs believe the objections have already been considered as far as is possible and reasonable, and the reviewers providing the objections can “live with” the decision, the draft decision becomes a formal decision of the Working Group. </li>
+					<li>If the substance of the objections was raised during discussion leading to the CfC, the chairs may give more consideration to those objections than to objections which were not raised during the discussion leading to the CfC, and reasonably could have been.</li>
```

Comment:
Same comment as above.

Reply:
Yeah, I think the last new bullet can go. If someone raises an objection that has already been covered and addressed, that should be easier to respond to adequately. I think that is covered by the first new bullet, and I support it.
However, if someone raises a wholly new concern, that _should_ get full attention. I get how you want folks to take part in discussion, but I think the first new bullet addresses that.